launcher/firstLaunch/firstlaunch_moc.cpp

```diff
@@ -158,6 +159,19 @@ void FirstLaunchView::exitSetup()
 		mainWindow->exitSetup();
 }
 
+// Initial platform-dependend settings
+void FirstLaunchView::setupPlatformSettings()
```

Comment:
Android currently uses separate launcher so this code won't work on Android.

I actually had same issue recently - ended up creating separate options ( "swipe" and "swipeDesktop") for desktop/mobile platforms, with desktop having set it to false by default and on mobile - set to true.

Ugly workaround, sure, but same can be said about this code. Not sure which option is better.

Reply:
> Android currently uses separate launcher so this code won't work on Android.

looking forward to solving that for 1.3, already have some initial work done

Reply:
We would probably still need possibility to defined default settings per platform - we already have cursor, map swiping/panning, and now UI for combat.

Will add this to my 1.3 todo-list, should be simple enough